faq/databases-for-postgresql-and-mysql.mdx

```diff
@@ -214,4 +214,10 @@ Examples:
 |-----------------	|-----------------------	|----------------	|------------------------------	|-----------------------------	|------------------	|
 | 192.168.0.0/24  	| 255.255.255.0         	| 24             	| 192.168.0.1                  	| 192.168.0.254               	| 192.168.0.33/24  	|
 | 10.18.0.0/16    	| 255.255.0.0           	| 16             	| 10.18.0.1                    	| 10.18.255.254               	| 10.18.21.42/16   	|
-| 192.168.18.0/26 	| 255.255.255.192       	| 26             	| 192.168.18.1                 	| 192.168.18.63               	| 192.168.18.37/26 	|
\ No newline at end of file
+| 192.168.18.0/26 	| 255.255.255.192       	| 26             	| 192.168.18.1                 	| 192.168.18.63               	| 192.168.18.37/26 	|
+
+## Can I use the second node on High Available Managed Database Instance ?
+
+No, the second node created when you activate the HA on your Managed Database Instance is a hot standby. This node allow fast failover in case of issue on the Primary Node.
```

Comment:
```suggestion
No, the second node created when you activate the HA on your Managed Database Instance is a hot standby. This node allows a fast failover in case of issues on the primary node.
```

Reply:
Solved